src/views/General/GeneralComponents/SidebarComponent.vue

```diff
@@ -44,40 +44,29 @@ const emit = defineEmits(['close-sidebar']);
       class="absolute top-16 bg-velvet transition-all duration-500 h-screen w-[300px] p-5 z-10"
       :class="{ 'left-0':  openSidebar, 'left-[-500px]': !openSidebar }"
     >
-      <section class="block bg-charcoal text-frostWhite w-full p-3 mx-auto my-3 text-center shadow-lg rounded-xl">
+      <section v-if="username" class="block bg-charcoal text-frostWhite w-full p-3 mx-auto my-3 text-center shadow-lg rounded-xl">
         <p>Logged in as: {{ username }}</p>
       </section>
-      <RouterLink @click="emit('close-sidebar')" to="/" class="block bg-frostWhite text-charcoal w-full p-3 mx-auto my-3 text-center shadow-lg rounded-xl hover:animate-pulse">
-          Welcome
+      <RouterLink v-if="username" @click="emit('close-sidebar')" to="/dashboard" class="block bg-frostWhite text-charcoal w-full p-3 mx-auto my-3 text-center shadow-lg rounded-xl hover:animate-pulse">
+        Dashboard
```

Comment:
Hooray! If we get around to adding an owner's dashboard, this might later need to be dynamic depending on the role of the user.